assets/js/homepage.js

```diff
@@ -18,8 +18,9 @@
 			return;
 		}
 
-		for ( var i = 0; i < homepageContent.querySelectorAll( '.entry-title, .entry-content' ).length; i++ ) {
-			homepageContent.querySelectorAll( '.entry-title, .entry-content' )[ i ].classList.add( 'loaded' );
+		var entries = homepageContent.querySelectorAll( '.entry-title, .entry-content' );
+		for ( var i = 0, i < entries.length; i++ ) {
```

Comment:
Thanks for applying the feedback @valdrinkoshi! Just one last comment, I think there is a typo:
```suggestion
		for ( var i = 0; i < entries.length; i++ ) {
```

Reply:
Argh, great catch! Fixed 👌